src/executionFiles/allowance.execute.ts

```diff
@@ -38,14 +38,14 @@ export const checkAllowance = async (
 
       if (new BigNumber(amount).gt(approved)) {
```

Comment:
IF you remove the chaincheck in the `StepExecutor.ts`, don't forget to add it here.

Reply:
this might be scope creep, as it is another issue: https://lifi.atlassian.net/browse/LF-126